scripts/lc-builds/toss3_hipcc4.1.0.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+BUILD_SUFFIX=lc_toss3-hipcc-4.1.0
```

Comment:
The builds scripts have been updated in develop to mirror the scripts in RAJA. So this one is inconsistent with the rest. Also, we should use HIP_HIPCC_FLAGS (which is supported in CMake directly) instead of RAJA_HIPCC_FLAGS (which will be reomoved.

Reply:
Makes sense to me, removed.